embark/workers/update/deps_host.sh

```diff
@@ -10,107 +10,214 @@
 # EMBArk is licensed under MIT
 #
 # Author(s): ClProsser, SirGankalot
-# Contributor(s): Luka Dekanozishvili, ashiven
+# Contributor(s): Luka Dekanozishvili, ashiven, Benedikt Kuehne
+#
+# Description: Prepares dependency packages on the host system for worker update
 
 set -e
 cd "$(dirname "$0")"
 
 if [[ ${EUID} -ne 0 ]]; then
-	echo "This script has to be run as root"
+	echo -e "\n[!!] ERROR: This script has to be run as root\n"
 	exit 1
 fi
 
-FILEPATH="$1"
-ZIPPATH="$2"
-VERSION="$3"
-DEPSCACHE="$4"
+echo -e "\n[+] Starting dependency package preparation script"
+echo -e "[*] Dependency Directory: $1"
```

Comment:
Style note: In EMBA we are enforcing curly brackets for variables: e.g. ${1}

Reply:
I agree. added to the TODO list